escher/src/text_input_helpers.cpp

```diff
@@ -5,18 +5,15 @@
 namespace TextInputHelpers {
 
 size_t CursorIndexInCommand(const char * text) {
-  size_t textLength = strlen(text);
-  if (textLength == 0) {
-    return 0;
-  }
-  for (size_t i = 0; i < textLength - 1; i++) {
+  size_t index = 0;
```

Comment:
you probably meant `i` here,  and forgot to increment it in the loop?

Reply:
Thanks! I'll just re-do the pull-request later :)

Reply:
Note that you can force push an edited commit into the branch and the PR will be updated automatically, so I don't think you need to have another PR :)